src/components/players/details/games/index.js

```diff
@@ -0,0 +1,73 @@
+import React from 'react';
+
+import Line from '../../../_shared/line';
+import Title from '../../../_shared/title';
+import Table from '../../../_shared/table';
+
+import styles from './index.module.scss';
+
+const Games = ({ teams, games }) => {
+  const gamesArray = Object.values(games);
+  const teamsArray = Object.values(teams);
+
+  const game = gamesArray.map(({
+    gameId,
+    isHomeGame,
+    hTeam,
+    vTeam,
+    gameDateUTC,
+    stats,
+    }) => {
+
+      const { points, assists, offReb, defReb, totReb } = stats;
+
+      const getGameResult = () => {
+        const playersTeam = isHomeGame ? hTeam : vTeam;
+        return playersTeam.isWinner ? 'W' : 'L';
+      };
+
+      const oppTeam = teamsArray.find(team => {
+        const oppTeamId = isHomeGame ? vTeam.teamId : hTeam.teamId;
+        return team.teamId === oppTeamId;
+      });
+
+      const score = parseInt(hTeam.score) > parseInt(vTeam.score) ?
+        `${hTeam.score} - ${vTeam.score}` :
+        `${vTeam.score} - ${hTeam.score}`;
+
+      const dateArray = new Date(gameDateUTC).toDateString().split(' ');
```

Comment:
This date manipulation should be inside of your `formattedDate` function, so that when you call the function on `line 57` it should be:

```
<div className={styles.date}>{formatUTCDate(gameDateUTC)}</div>
```

Note, the renamed function.